packages/client/src/utils/helpers.ts

```diff
@@ -100,3 +102,74 @@ export const replaceHTMLTags = (template: string) => {
   const regex = new RegExp(/( |<([^>]+)>)/gi);
   return template.replaceAll(regex, "");
 };
+
+/**
+ * Inserts a placeholder value into an input field and updates the corresponding Formik field.
+ * @param setFieldValue - The Formik `setFieldValue` function to update the field value.
+ * @param input - Reference to the input element.
+ * @param type - The type of template ie email or sms
+ * @returns A function that takes a button value and inserts it into the input field.
+ */
+
+export const insertValuePlaceholder =
+  (
+    setFieldValue: (
+      field: string,
+      value: any,
+      shouldValidate?: boolean | undefined
+    ) => Promise<void | FormikErrors<OrganizationData>>,
```

Comment:
Since you know the shape of the form, you could do something like this: 
```ts
const insertValuePlaceholder = <F extends keyof OrganizationData>(
  field: F, // Constraining field to keys of OrganizationData
  value: OrganizationData[F], // Not using 'any' 
   ...) => Promise<void | FormikErrors<OrganizationData>>
```